helm/cluster-vsphere/values.schema.json

```diff
@@ -500,6 +503,92 @@
 						}
 					}
 				},
+				"nodeClasses": {
+					"type": "object",
+					"title": "Template to define control plane nodes",
+					"properties": {
+						"default": {
+							"type": "object",
+							"title": "Default node class configuration.",
+							"additionalProperties": false,
+							"properties": {
+								"cloneMode": {
+									"type": "string",
+									"title": "Template clone mode",
+									"description": "VM template cloning method.",
+									"default": "linkedClone"
+								},
+								"diskGiB": {
+									"type": "integer",
+									"title": "Disk size",
+									"description": "Control plane node root volume size, in GB.",
+									"default": 50,
```

Comment:
I'm not sure we should set a default here.
If the base disk of the VM template is <50GB and that value isn't set, it will break the FS and the node won't deploy properly.
Leaving it empty will just clone the VM template with the current disk size.

Reply:
that makes sense, I'm going to keep the min size though as we need to enforce some basic limits